feeed/entropies.py

```diff
@@ -1,6 +1,10 @@
 import inspect
 import os
 import subprocess
+from collections import Counter
+import math
+from Levenshtein import distance 
```

Comment:
please add new packages to https://github.com/lmu-dbs/feeed/blob/5e062e9c839d8943bd1950347598e08e2cc3ae99/setup.py#L11

Reply:
- Both Counter and math are default Python classes so there's no need to install them separately. 
- Updated the Levenshtein package